plonk/src/circuit/gates.rs

```diff
@@ -44,6 +44,18 @@ pub trait Gate<F: Field>: Downcast + GateClone<F> {
     fn q_lookup(&self) -> F {
         F::zero()
     }
+    /// UltraPlonk lookup domain separation selector.
+    fn q_dom_sep(&self) -> F {
+        F::zero()
+    }
+    /// UltraPlonk table keys.
+    fn table_key(&self) -> F {
+        F::zero()
+    }
+    /// UltraPlonk table domain sepration ids
```

Comment:
sepration -> separation

Reply:
Thanks, fixed in f8cf2f64504edf04467fd0704cb1f190c10f085f